certbot/docs/install.rst

```diff
@@ -254,7 +254,7 @@ If you want to uninstall ``certbot-auto``, you can follow our instructions
 Problems with Python virtual environment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When using ``certbot-auto`` on a low memory system such as VPS with less than
+When using ``certbot`` on a low memory system such as VPS with less than
```

Comment:
This is in a subsection under `certbot-auto`. The table of contents for this file looks like
![Screen Shot 2021-03-24 at 4 06 49 PM](https://user-images.githubusercontent.com/6504915/112395075-fa88b100-8cba-11eb-91ac-c45bcc1fac02.png).

Because of this, I think we should still reference certbot-auto here if we keep these docs.

The only reason I can think of for keeping them is to not break links such as https://github.com/certbot/certbot/blob/92a66454b6ece097a987621031b16b1fb2f490fd/certbot-auto#L1725, however, I think this particular section about problems with the virtual environment is largely outdated since cryptography started shipping precompiled wheels many years ago.

I think my personal preference is to delete every reference to certbot-auto in this file, but I don't have a strong preference about what we keep/delete here for now.

Reply:
ACK no I definitely agree we should remove the whole section; I did NOT think to grep for Certbot-Auto! What!

Reply:
Ok, this is the only instance of Certbot-Auto/Certbot-auto/Letsencrypt-auto/Letsencrypt-Auto

Reply:
ohhhh no jk, I kept this because it said "we used to have", and now points to the uninstall instructions. But yeah we don't need the python virtual environment anymore; if they have issues, they should try a different installation method.

Reply:
I think keeping the certbot-auto section for the link to the uninstall section is fine, however, in case you're not aware, our instruction generator links to it to with the text
![Screen Shot 2021-03-24 at 4 32 08 PM](https://user-images.githubusercontent.com/6504915/112397027-7df7d180-8cbe-11eb-970c-cfa3076b75c1.png)

Reply:
We were discussing somewhere else about getting rid of this entire page since it duplicates the instruction generator. But I think if we're going to keep it around, it makes sense to have that section and link to the uninstall here for now.